docs/01-overview/main-areas/application-connectivity/ac-05-useful-links.md

```diff
@@ -34,20 +34,20 @@ If you're interested in learning more about the Application Connectivity area, f
 - Analyze Application Connectivity specification and configuration files:
 
```

Comment:
The `Disable SSL certificate verification` link (line 13) is broken. Gosia moved it to tutorials and must have forgotten to update the link. Would you be so kind and update it here? 
It should be `../../../03-tutorials/00-application-connectivity/ac-14-disable-ssl-certificate-verification.md`.